src/Assert.php

```diff
@@ -168,6 +168,14 @@
  */
 class Assert
 {
+    /**
+     * String.
+     *
+     * @param        $value
```

Comment:
For cases like this, I would suggest one of the following cases:
* Add the annotation as `@param mixed $value`
* Be consistent with the check that we're expecting to validate - for this case `@param string $value`